.github/workflows/tests.yml

```
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        branch: [origin/main, 3.11, '3.10', 3.9, 3.8, 3.7]
+        branch: [origin/main, 3.12, 3.11, '3.10', 3.9, 3.8, 3.7]
```

Should 3.7 be dropped, now it's EOL?

```suggestion
        branch: [origin/main, 3.12, 3.11, '3.10', 3.9, 3.8]
```

---

Yep :)